Script/TypeScript/Atomic.d.ts

```diff
@@ -318,6 +318,30 @@ declare module Atomic {
    export var RAY_TRIANGLE_UV: RayQueryLevel;
 
 
+   // enum RenderCommandType
```

Comment:
The thinking on Atomic.d.ts being in the repo is that you would be able to use it to compile without building the engine, though I think that should probably be ignored too.  I am thinking more of packaging node with the repo which is probably pretty easy to do